rules/style.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
         'import/no-dynamic-require': 'off',
         'import/no-extraneous-dependencies': 'off',
         'import/no-unresolved': 'off',
+        'jsx-a11y/click-events-have-key-events': 'off',
```

Comment:
I think this should go to `react.js` rather than `style.js`, because this rule requires the JSX environment variable to be enabled within ESLint.

For the "legacy" endpoint of our ESLint config we don't have that variable enabled though. I might be wrong, and everything will continue to work once we use this version, but just putting this out here.